### PR TITLE
fix lighthouse `unsized-img` test for meshery page

### PR DESCRIPTION
**Pull request by ayushthe1**
**Description**

This PR fixes the failing `unsized-img` test for meshery page

**Notes for Reviewers**
Changes must be first checked on safari browser before merging

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit c393d2ea80cbfc653ff828d74348fe4b95d1c42b at: https://641250dfb46317280a20d616--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 399ecccf6512390c0cd1377f8c4bb3922d92af9b at: https://6414b581cfc66b168c3b7a92--layer5.netlify.app

File: src/sections/Meshery/Features-section/index.js [hunk 1]
```diff
@@ -68,13 +68,13 @@ const FeaturesSection = () => {
             <Col xs={12} xl={8} className="smp-section-caraousel">
               <Slider {...settings}>
                 <div>
-                  <img src={Slide1} alt="Slide 1" loading="lazy" />
+                  <img src={Slide1} alt="Slide 1" width="1918px" height="978px" loading="lazy" />
```
Comment (Nikhil-Ladha) [inline src/sections/Meshery/Features-section/index.js:71]: Based on what are we setting these values?

**Comment by Chadha93** [comment]
@ayushthe1 Let's discuss this on the Websites call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#) if you would. :)

File: src/sections/Meshery/Features-section/index.js (hunk 1 above)
Comment (ayushthe1) [reply src/sections/Meshery/Features-section/index.js:71]: @Nikhil-Ladha These are the original size of the images. However, this change doesn't matter as I have made a new PR #3978 which replaces this img(gif) with a video and sets height, width = 100%

**Pull request closed without merging**